source/2020-01-31-the-ember-times-issue-133.md

```diff
@@ -0,0 +1,211 @@
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [Getting into writing Ember posts ✏️](https://dev.to/t/ember/latest)
+
+This week, we give a hearty shout-out to [Ilya Radchenko (@knownasilya)](https://github.com/knownasilya) and [Michal Bryxí (@MichalBryxi)](https://github.com/MichalBryxi). Since January, they have been regularly sharing their learnings of Ember through short blog posts.
+
+Here are some of our favorites:
+
+- [Ember Components at Their Simplest](https://ilyaradchenko.com/ember-components-at-their-simplest/)
+- [When Components Aren't Enough](https://ilyaradchenko.com/when-components-aren't-enough/)
+- [6 Tips for Transitioning to Ember CSS Modules](https://ilyaradchenko.com/6-tips-for-transitioning-to-ember-css-modules/)
+- [Add Fulltext Filter Support to ember-cli-mirage](https://dev.to/michalbryxi/add-fulltext-filter-support-to-ember-cli-mirage-3nef)
+- [Shining Some Light on Timing Out Tests](https://dev.to/michalbryxi/shining-some-light-on-timing-out-tests-35e6)
+- [Component Arguments vs HTML Attributes Bug Hunt](https://dev.to/michalbryxi/component-arguments-vs-html-attributes-bug-hunt-1ke9)
+
+We encourage you to have a read. You may be pleasantly surprised to see that, through small steps, one can end up teaching and helping others a lot. 💞
+
+---
+
+## [Section title in sentence case 🐹](#section-url)
+
+<change section title emoji>
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [Section title in sentence case 🐹](#section-url)
+
+<change section title emoji>
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [Community feedback from FlashRecruit 👏](https://flashrecruit.com)
+
+At [**FlashRecruit**](https://flashrecruit.com), we're building a conversational job marketplace. We've been using Ember Octane since before the official preview--don't tell anyone--to build the new version of our job board and dashboard.
+
+Ember Octane allowed us to quickly prototype an application, move on to production with little tech debt, and feel well supported throughout the process.
+
+<!--alex ignore nuts-->
+Tracked Properties and Glimmer Components provide a wonderful mental model to work from. They also simplify our code by bringing it closer to JavaScript and HTML. We had a new hire that only had to learn the Octane way, which ended up far smoother than the classic-style onboarding.
+
+<!--alex ignore period-->
+We saw a few issues in the preview period, but the community was quick to help us figure out how to move past them. They were mostly fixed by the official release, as well.
+
+<!--alex ignore just-->
+We look forward to the enhanced tracking changes coming in RFCs like [#577](https://github.com/emberjs/rfcs/pull/577). Honestly, anything beyond this point is just gravy.
+
```

Comment:
@amyrlam Before merging, can we add a paragraph asking readers to submit their team story? Here's [what we had written for Outdoorsy](https://github.com/ember-learn/ember-blog/pull/459/files#diff-9aebd6de851062be3f1b199a02c7df2aR158) for an idea. Thanks!

Reply:
@ijlee2 thanks, great idea! will add a commit